src/frontend/i18n/locales/es-ES.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
How did you chose the locales list ?

At this time, according to our [Crowdin project ](https://crowdin.com/project/ashleyforum), the source language is English and the project is translated to French (fr_FR) only. I'll be happy to see Ashley translated in other languages, but I think we should be consistent between the backend and the frontend translations.

Reply:
I only left french file